check.go

```diff
@@ -51,8 +51,8 @@ type CustomEventCheckFragment struct {
 }
 
 type ServiceOwnershipCheckFragment struct {
-	RequireContactMethod *bool        `graphql:"requireContactMethod"`
-	ContactMethod        *ContactType `graphql:"contactMethod"`
+	RequireContactMethod *bool                             `graphql:"requireContactMethod"`
```

Comment:
I'll defer to Kyle since I'm still not 100% clear on what needs to be a pointer or not but I think we've only set them on the Create and Update types for other implementations.

Reply:
It needs to be a pointer if the graphql field is "allowed to be nullable"